### ISSUE TITLE
sek

### ISSUE THREAD

**Issue by ranmengying** [ggsDing/Bi-SRNet#34]
Hello author, thank you very much for the code, I trained my own dataset with your SSCD-L model, and evaluated it with the trained weight, but the sek=43.62 in the weight, and the evaluated sek=5.42, I would like to ask you if this result is correct, if not, do you know where the error is. The training set is divided into a training set, a validation set, and a test set

**Comment by DingLei14** [comment]
That seems abnormal. Check if your training and testing data have overlap
